doc/howto/network_bridge_firewalld.md

```diff
@@ -1,52 +1,120 @@
-(network-bridge-firewalld)=
-# How to configure Firewalld
+---
+discourse: 10034,9953
+---
 
-## Allow DHCP, DNS with Firewalld
+(network-bridge-firewall)=
+# How to configure your firewall
 
-In order to allow instances to access the DHCP and DNS server that LXD runs on the host when using firewalld
-you need to add the host's bridge interface to the `trusted` zone in firewalld.
+If you run a firewall on your system, you might need to configure it to allow network traffic between the managed LXD bridge and the host.
+Otherwise, some network functionality (DHCP, DNS and external network access) might not work as expected.
 
-To do this permanently (so that it persists after a reboot) run the following command:
+By default, managed LXD bridges add firewall rules to ensure full functionality.
+(This behavior can be controlled through the `ipv4.firewall` or `ipv6.firewall` {ref}`configuration options <network-bridge-options>`.)
+However, depending on the firewall subsystem that you use, firewall rules added by other applications might interfere with the firewall rules that LXD adds.
 
-```
-firewall-cmd --zone=trusted --change-interface=<LXD network name> --permanent
-```
+See the following sections for instructions on how to properly configure firewalld and UFW, respectively.
 
-E.g. for a bridged network called `lxdbr0` run the command:
+## Allow traffic from/to the LXD bridge
 
-```
-firewall-cmd --zone=trusted --change-interface=lxdbr0 --permanent
-```
+To allow instances to access the DHCP and DNS server that LXD runs on the host, you must configure your firewall to allow network traffic from and to the LXD bridge.
```

Comment:
@ru-fu the use of "from and to" in this line didn't read quite right to me (I am more used to saying "to and from" instead). Neither are wrong of course, but I was wondering if this ordering felt more natural to you?

Reply:
We can avoid the subject by saying something like "to allow network traffic between the instances and the LXD bridge" :)

Reply:
I originally had a mixture of "to and from" and "from and to" and just went for one of the options to be consistent. ;) But the choice was just random.

Reply:
Yeah its not a problem at all, just felt a little odd :)

Reply:
And that is very good for me to know - you're the native speaker. :)